doc/release-notes/release-notes-4.0.0.md

```diff
@@ -0,0 +1,41 @@
+zend v4.0.0
+=========
+
+As the previous version of `zend` (3.3.1), this RC3 is expected to deprecate at block **#1359120**, which will occur on **May 31st, 2023 at approximately 12:00 PM UTC**. Please, update to `v4.0.0` before May 31st (the final release will be available soon).
+
+A **hard fork** has activated on testnet at block **#1228700**, which has occurred on **March 13th at 14:37:43 GMT+0000**.
```

Comment:
Let's take the chance to update the release notes as well, I suggest something like this, but feel free to edit:

```suggestion
Zend `3.3.1` is going to deprecate at block **#1359120**, which will occur on **May 31st, 2023 at approximately 12:00 PM UTC**. Please, update to `v4.0.0` before May 31st.

Zend `4.0.0` is going to deprecate at block **#1422426**, which will occur on **September 19th, 2023 at approximately 12:00 PM UTC**.

A **hard fork** is going to activate on mainnet at block **#1363115**, which will occur on **June 7th, 2023 at approximately 13:00 UTC**.

The same **hard fork** has activated on testnet at block **#1228700**, which has occurred on **March 13th at 14:37:43 GMT+0000**.
```